Makefile

```diff
@@ -447,10 +447,18 @@ ui-test-general:
 	$(NPM_BIN) run --prefix awx/ui pretest
 	$(NPM_BIN) run --prefix awx/ui/ test-general --runInBand
 
+## Create empty ui/build for headless builds
+ui-headless:
+	@echo "=== Creating empty awx/ui/build ==="
+	@rm -rf awx/ui/build/static
+	@mkdir -p awx/ui/build/static
+	@touch awx/ui/build/static/HEADLESS
+	@echo "=== Done creating empty awx/ui/build ==="
+
 # NOTE: The make target ui-next is imported from awx/ui_next/Makefile
 HEADLESS ?= no
 ifeq ($(HEADLESS), yes)
-dist/$(SDIST_TAR_FILE):
+dist/$(SDIST_TAR_FILE): ui-headless ui-next/headless
```

Comment:
Should we make ui-next/headless a target of ui-headless so that if I run `make ui-headless` from the root folder it will hit both targets?